aws/sdk/integration-tests/Cargo.toml

```diff
@@ -8,6 +8,7 @@ members = [
     "iam",
     "kms",
     "lambda",
+    "no-default-features",
```

Comment:
Fine for now, but this will need a follow up to actually run this in CI since CI only runs integration tests named after services.

Reply:
Issue created: https://github.com/awslabs/smithy-rs/issues/2056